### ISSUE TITLE
IstioControlPlane global.defaultTolerations not working

### ISSUE THREAD

**Issue by elvizlai** [istio/istio#19605]
**Bug description**

```
helm template install/kubernetes/helm/istio \
    --set global.defaultTolerations[0].key="node-role.kubernetes.io/master" \
    > $HOME/istio.yaml
```
~~NOT work now, any break change?~~

**SOLVED: changing `oh-my-zsh` to `bash` works.**

Then I tried using `istioctl` with `IstioControlPlane`
```
apiVersion: install.istio.io/v1alpha2
kind: IstioControlPlane
spec:
  security:
    components:
      certManager:
        enabled: true
  values:
    global:
      defaultTolerations:
        - key: "node-role.kubernetes.io/master"
          operator: "Equal"
          effect: "PreferNoSchedule"
```

It's output:
```
input file failed validation with the following errors: unknown field "defaultTolerations" in v1alpha1.GlobalConfig
```

[ ] Configuration Infrastructure
[ ] Docs
[X ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Expected behavior**

**Steps to reproduce the bug**

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**
v1.4.2

helm:
```
Client: &version.Version{SemVer:"v2.14.2", GitCommit:"a8b13cc5ab6a7dbef0a58f5061bcc7c0c61598e7", GitTreeState:"clean"}
```

**How was Istio installed?**

**Environment where bug was observed (cloud vendor, OS, etc)**

**Comment by morvencao** [comment]
@elvizlai Which version are you using? I didn't reproduce this in master:
```
# helm template install/kubernetes/helm/istio \
>     --set global.defaultTolerations[0].key="node-role.kubernetes.io/master" \
>     > $HOME/istio.yaml
# kubectl apply -f $HOME/istio.yaml
poddisruptionbudget.policy/istio-galley created
poddisruptionbudget.policy/istio-ingressgateway created
poddisruptionbudget.policy/istio-policy created
poddisruptionbudget.policy/istio-telemetry created
poddisruptionbudget.policy/istio-pilot created
poddisruptionbudget.policy/istio-citadel created
poddisruptionbudget.policy/istio-sidecar-injector created
configmap/istio-galley-configuration created
configmap/prometheus created
configmap/istio-security-custom-resources created
configmap/istio created
configmap/istio-sidecar-injector created
serviceaccount/istio-galley-service-account created
serviceaccount/istio-ingressgateway-service-account created
serviceaccount/istio-mixer-service-account created
serviceaccount/istio-pilot-service-account created
serviceaccount/prometheus created
serviceaccount/istio-security-post-install-account created
clusterrole.rbac.authorization.k8s.io/istio-security-post-install-default created
clusterrolebinding.rbac.authorization.k8s.io/istio-security-post-install-role-binding-default created
job.batch/istio-security-post-install-latest created
serviceaccount/istio-citadel-service-account created
serviceaccount/istio-sidecar-injector-service-account created
serviceaccount/istio-multi created
clusterrole.rbac.authorization.k8s.io/istio-galley-default created
clusterrole.rbac.authorization.k8s.io/istio-mixer-default created
clusterrole.rbac.authorization.k8s.io/istio-pilot-default created
clusterrole.rbac.authorization.k8s.io/prometheus-default created
clusterrole.rbac.authorization.k8s.io/istio-citadel-default created
clusterrole.rbac.authorization.k8s.io/istio-sidecar-injector-default created
clusterrole.rbac.authorization.k8s.io/istio-reader created
clusterrolebinding.rbac.authorization.k8s.io/istio-galley-admin-role-binding-default created
clusterrolebinding.rbac.authorization.k8s.io/istio-mixer-admin-role-binding-default created
clusterrolebinding.rbac.authorization.k8s.io/istio-pilot-default created
clusterrolebinding.rbac.authorization.k8s.io/prometheus-default created
clusterrolebinding.rbac.authorization.k8s.io/istio-citadel-default created
clusterrolebinding.rbac.authorization.k8s.io/istio-sidecar-injector-admin-role-binding-default created
clusterrolebinding.rbac.authorization.k8s.io/istio-multi created
role.rbac.authorization.k8s.io/istio-ingressgateway-sds created
rolebinding.rbac.authorization.k8s.io/istio-ingressgateway-sds created
service/istio-galley created
service/istio-ingressgateway created
service/istio-policy created
service/istio-telemetry created
service/istio-pilot created
service/prometheus created
service/istio-citadel created
service/istio-sidecar-injector created
deployment.apps/istio-galley created
deployment.apps/istio-ingressgateway created
deployment.apps/istio-policy created
deployment.apps/istio-telemetry created
deployment.apps/istio-pilot created
deployment.apps/prometheus created
deployment.apps/istio-citadel created
deployment.apps/istio-sidecar-injector created
...
```

**Comment by elvizlai** [comment]
> @elvizlai Which version are you using? I didn't reproduce this in master:
> 
> ```
> # helm template install/kubernetes/helm/istio \
> >     --set global.defaultTolerations[0].key="node-role.kubernetes.io/master" \
> >     > $HOME/istio.yaml
> # kubectl apply -f $HOME/istio.yaml
> poddisruptionbudget.policy/istio-galley created
> poddisruptionbudget.policy/istio-ingressgateway created
> poddisruptionbudget.policy/istio-policy created
> poddisruptionbudget.policy/istio-telemetry created
> poddisruptionbudget.policy/istio-pilot created
> poddisruptionbudget.policy/istio-citadel created
> poddisruptionbudget.policy/istio-sidecar-injector created
> configmap/istio-galley-configuration created
> configmap/prometheus created
> configmap/istio-security-custom-resources created
> configmap/istio created
> configmap/istio-sidecar-injector created
> serviceaccount/istio-galley-service-account created
> serviceaccount/istio-ingressgateway-service-account created
> serviceaccount/istio-mixer-service-account created
> serviceaccount/istio-pilot-service-account created
> serviceaccount/prometheus created
> serviceaccount/istio-security-post-install-account created
> clusterrole.rbac.authorization.k8s.io/istio-security-post-install-default created
> clusterrolebinding.rbac.authorization.k8s.io/istio-security-post-install-role-binding-default created
> job.batch/istio-security-post-install-latest created
> serviceaccount/istio-citadel-service-account created
> serviceaccount/istio-sidecar-injector-service-account created
> serviceaccount/istio-multi created
> clusterrole.rbac.authorization.k8s.io/istio-galley-default created
> clusterrole.rbac.authorization.k8s.io/istio-mixer-default created
> clusterrole.rbac.authorization.k8s.io/istio-pilot-default created
> clusterrole.rbac.authorization.k8s.io/prometheus-default created
> clusterrole.rbac.authorization.k8s.io/istio-citadel-default created
> clusterrole.rbac.authorization.k8s.io/istio-sidecar-injector-default created
> clusterrole.rbac.authorization.k8s.io/istio-reader created
> clusterrolebinding.rbac.authorization.k8s.io/istio-galley-admin-role-binding-default created
> clusterrolebinding.rbac.authorization.k8s.io/istio-mixer-admin-role-binding-default created
> clusterrolebinding.rbac.authorization.k8s.io/istio-pilot-default created
> clusterrolebinding.rbac.authorization.k8s.io/prometheus-default created
> clusterrolebinding.rbac.authorization.k8s.io/istio-citadel-default created
> clusterrolebinding.rbac.authorization.k8s.io/istio-sidecar-injector-admin-role-binding-default created
> clusterrolebinding.rbac.authorization.k8s.io/istio-multi created
> role.rbac.authorization.k8s.io/istio-ingressgateway-sds created
> rolebinding.rbac.authorization.k8s.io/istio-ingressgateway-sds created
> service/istio-galley created
> service/istio-ingressgateway created
> service/istio-policy created
> service/istio-telemetry created
> service/istio-pilot created
> service/prometheus created
> service/istio-citadel created
> service/istio-sidecar-injector created
> deployment.apps/istio-galley created
> deployment.apps/istio-ingressgateway created
> deployment.apps/istio-policy created
> deployment.apps/istio-telemetry created
> deployment.apps/istio-pilot created
> deployment.apps/prometheus created
> deployment.apps/istio-citadel created
> deployment.apps/istio-sidecar-injector created
> ...
> ```

opts... I changed my oh-my-zsh to bash, now works.

**Comment by elvizlai** [comment]
Friendly ping, this still not available in release `1.4.5`
